apps/balancer-tools/src/app/apr/api/route.ts

```diff
@@ -0,0 +1,69 @@
+import { NextRequest, NextResponse } from "next/server";
+
+import votingGauges from "#/data/voting-gauges.json";
+
+import { calculatePoolStats } from "../(utils)/calculatePoolStats";
+import { Round } from "../(utils)/rounds";
+
+export async function GET(request: NextRequest) {
+  const poolId = request.nextUrl.searchParams.get("poolid");
+  const roundId = request.nextUrl.searchParams.get("roundid");
+  if (!roundId && !poolId) {
+    return NextResponse.json({ error: "no round or poolId provided" });
+  }
+
+  if (roundId && poolId) {
+    const poolStats = await calculatePoolStats({ roundId, poolId });
+
+    return NextResponse.json(poolStats);
```

Comment:
ok, so apparently this is already [cached by default](https://nextjs.org/docs/app/building-your-application/routing/route-handlers#caching) 👍